config.yaml

```diff
@@ -15,6 +15,28 @@ options:
       to accept the charm default. Choosing 'auto' will install the latest 
       supported version of etcd at deploy time, but will not automatically upgrade
       to a newer version thereafter.
+  heartbeat_interval:
+    type: int
+    default: 100
+    description: |
+      The frequency with which the leader will notify followers that it is
+      still the leader. By default, etcd uses a 100ms heartbeat interval.
```

Comment:
This default matches the docs

> **--heartbeat-interval**
    Time (in milliseconds) of a heartbeat interval.
    default: “100”
    env variable: ETCD_HEARTBEAT_INTERVAL